app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,35 @@
+class MoviesController < ApplicationController
+  respond_to :json
+
+  def index
+    user = User.find(params[:user_id])
+    movies = user.movies
+    data = { movies: movies }
+    render json: data, status: 200
+  end
+
+  def create
+    @user = User.find(params[:user_id])
+    @movie = @user.movies.new(movie_params)
+
+    if @user && @movie.save
+      data = { id: @movie.id, title: @movie.title, overview: @movie.overview, release_date: @movie.release_date, user_id: @movie.user_id }
+      render json: data
+    else
+      data = { errors: @movie.errors.messages }
+      render json: data, status: 422
+    end
+  end
+
+  def destroy
+    user = User.find(params[:user_id])
```

Comment:
try doing this in 1 line with active record:  https://apidock.com/rails/ActiveRecord/QueryMethods/includes